csharp/src/Ice/TcpAcceptor.cs

```diff
@@ -135,8 +144,9 @@ internal TcpAcceptor(TcpEndpoint endpoint, TransportInstance instance, string ho
             }
         }
 
+        private readonly string _adapterName;
```

Comment:
I assume you plan to move these fields in a future PR?

Reply:
Yes we can cleanup this in a future PR, easier to review that way